packages/driver/src/cypress/error_messages.js

```diff
@@ -931,23 +931,29 @@ module.exports = {
         You passed: ${format(staticResponse)}`, 8)
     },
     intercept: {
+      extra_arguments: ({ argsLength, overload }) => {
+        return cyStripIndent(`\
+          ${cmd('intercept', overload.join(', '))} accepts a maximum of ${overload.length} arguments, but ${argsLength} arguments were passed.
+          
+          Did you intend to call a different \`cy.intercept()\` overload?`, 10)
```

Comment:
So in my case this reads as 

>cy.intercept(url, handler?) accepts a maximum of 2 arguments, but 3 arguments were passed.

I guess if I read very closely I would see you're refering to this exact signature accepting only 2 args, but I initially read this as 'cy.intercept() has max of 2 args' and I was like...that's not true.

I'm not sure what this last sentence is intending to say. We don't use the word 'overload' anywhere in the intercept docs, so what is overload's meaning here? 

It's a tough error to write.

**One Suggestion:**

The error has a link to docs in a 'Learn more' link, so I think it's ok to reference.

>The cy.intercept(url, handler?) signature accepts a maximum of 2 arguments, but 3 arguments were passed.
>
>Please refer to the docs for all accepted signatures for cy.intercept().

Reply:
Overloading: https://en.wikipedia.org/wiki/Function_overloading

I like your error more though, I feel like "signature" is a way better word.

Reply:
Yah I guess I'm used to seeing the term as 'overloading' more than the noun 'overload'.

Reply:
Updated.